quickstart/src/main/java/com/twilio/video/quickstart/util/CameraCapturerCompat.java

```diff
@@ -115,4 +131,25 @@ private String getCameraId(CameraCapturer.CameraSource cameraSource) {
             return backCameraPair.first;
         }
     }
+
+    private boolean isLollipopApiSupported() {
+        return Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP;
+    }
+
+    private boolean isPrivateImageFormatSupportedForCameraId(String cameraId) {
+        boolean isPrivateSupported = true;
```

Comment:
Let's default to false in case an exception is thrown while accessing the characteristics.

Reply:
I like the name of the method. Let's use the same convention in the variable.

`boolean isPrivateImageFormatSupported = false;`